### PR TITLE
issue_templates added in .github

### LINKED ISSUE #4: docs: Add an Issue template
## Description 
Add a the issue template which will help the newcomers to open any issue in a particular manner

### PR DESCRIPTION
**Pull request by luckykumarirai**
I have added the issue templates in the  .github folder.
close #4

### CODE REVIEW

File: periodo/.github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```
Comment (khushishikhu) [inline periodo/.github/ISSUE_TEMPLATE/bug.md:5]:
```suggestion

```

File: periodo/.github/ISSUE_TEMPLATE/bug.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+assignees: ''
```
Comment (khushishikhu) [inline periodo/.github/ISSUE_TEMPLATE/bug.md:6]:
```suggestion

```

File: periodo/.github/ISSUE_TEMPLATE/bug.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```
Comment (khushishikhu) [inline periodo/.github/ISSUE_TEMPLATE/bug.md:11]: Comment out the statement in every description part of the Issue template

**Review by khushishikhu — CHANGES_REQUESTED** [review]

**Comment by Aaishpra** [comment]
@luckykumarirai can you add the suggested changes so that this PR can be merged

**Comment by luckykumarirai** [comment]
sure I will

**Review by khushishikhu — APPROVED** [review]

**Comment by khushishikhu** [comment]
@luckykumarirai LGTM!! Thank you so much for making the template. Keep contributing.

**Pull request merged**